saleor/graphql/schema.graphql

```diff
@@ -6132,6 +6132,7 @@ enum WebhookEventTypeEnum {
   CHECKOUT_CREATED
   CHECKOUT_UPDATED
   FULFILLMENT_CREATED
+  FULFILLMENT_CANCELLED
```

Comment:
It's a detail, but the status is called `FulfillmentStatus.CANCELED`, so we could rename it wherever it is used in this PR accordingly.